tests/piqueserver/test_config.py

```diff
@@ -137,7 +137,7 @@ def test_fail_load(self):
 
     def test_json(self):
         config = ConfigStore()
-        f = 'piqueserver/config/config.json'
+        f = 'piqueserver/config/simple.json'
```

Comment:
yeah, I see. I don't think this should go in the config folder as that would mean it gets copied over with `--copy-config`. Ideally, just use StringIO